cross/python3/Makefile

```diff
@@ -82,12 +82,14 @@ python3_custom_install:
 python3_post_install: $(WORK_DIR)/python-cc.mk
 	mkdir -p $(PYTHON_LIB_CROSS)
 	cp -R $(HOSTPYTHON_LIB_NATIVE) $(PYTHON_LIB_CROSS)/../
+	@$(RUN) $(PYTHON_NATIVE) -m crossenv $(STAGING_INSTALL_PREFIX)/bin/python$(PKG_VERS_MAJOR).$(PKG_VERS_MINOR) $(WORK_DIR)/crossenv/
```

Comment:
So here we basically run `crossenv` to setup the `activate` script at `$(WORKDIR)/crossenv/bin/activate` using the native python script and targetting the cross compiled python installation.